.circleci/config.yml

```diff
@@ -1,22 +1,161 @@
 version: 2.1
 
-jobs:
-  test:
+commands:
+  install_dependencies:
+    description: "Install development dependencies."
+    steps:
+      - run: composer install
+  mkdir_artifacts:
+    description: "Make Artifacts directory"
+    steps:
+      - run:
+          command: |
+            [ ! -d "/tmp/artifacts" ] && mkdir /tmp/artifacts &>/dev/null
+  svn_setup:
+    description: "Setup SVN"
+    steps:
+      - mkdir_artifacts
+      - run:
+          command: |
+            cd /tmp/artifacts
+            SLUG=grep 'Text Domain:' /tmp/src/plugin.php | awk -F: '{print $2}' | sed 's/^\s//'
```

Comment:
@BMO-tech Just a heads up that not all of our plugins have a `plugin.php` in the root. Some use the `[plugin-slug].php` format instead: https://github.com/studiopress/simple-social-icons

If we plan to reuse this config with minimal modifications, is it worth having a `PLUGIN_ROOT_FILENAME` or similar env var? (I notice the version is fetched from that file too, hence me suggesting `PLUGIN_ROOT_FILENAME` instead of just `PLUGIN_SLUG`.)

Reply:
I think it will be safer to slightly alter each `.circleci/config.yml` file per plugin to reference the right "`plugin.php`" or equivalent file. The less we have to manage in CircleCI itself the better.

Reply:
I'm good with that as long as we remember. :-) Probably only have to set it once for each plugin anyway.